### PR TITLE
ACN_CD_QualysVM_ConnecterFix

### PR DESCRIPTION
**Pull request by chicduong**
Fixes #

## Proposed Changes

  - Increase default timer trigger to 60 mins
  - Added host.json to increase execution timeout from 5m to 10m
  - Updated the data connector json, azure deploy json, and Azure Function zip accordingly

### CODE REVIEW

File: DataConnectors/Qualys VM/host.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "version": "2.0",
+    "functionTimeout": "00:10:00",
+    "managedDependency": {
+      "Enabled": true
+    },
+    "extensionBundle": {
+      "id": "Microsoft.Azure.Functions.ExtensionBundle",
+      "version": "[1.*, 2.0.0)"
```
Comment (preetikr) [inline DataConnectors/Qualys VM/host.json:9]: The opening bracket [  and closing parenthesis  ) - are not matching

File: DataConnectors/Qualys VM/host.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (preetikr) [inline DataConnectors/Qualys VM/host.json:1]: Shouldn't there be corresponding changes in run.ps1 to use the new properties defined here? 
What does version = 2.0 signify here?

Comment (chicduong) [reply DataConnectors/Qualys VM/host.json:1]: This is from a MSFT documentation reference.

https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json

File: DataConnectors/Qualys VM/host.json (hunk 1 above)
Comment (chicduong) [reply DataConnectors/Qualys VM/host.json:9]: This is taken from the sample from the MFST documentation and also compared against out data connectors (Zoom) 

https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json

**Pull request merged**